lib/spells/effects/Timed.cpp

```diff
@@ -175,16 +181,17 @@ void Timed::prepareEffects(SetStackEffect & sse, const Mechanics * m, const Effe
 		if(describe)
 			describeEffect(sse.battleLog, m, converted, affected);
 
-		si32 power = 0;
+		const auto tier = std::max(affected->creatureLevel(), 1); //don't divide by 0 for certain creatures (commanders, war machines)
 
 		//Apply hero specials - peculiar enchants
-		const auto tier = std::max(affected->creatureLevel(), 1); //don't divide by 0 for certain creatures (commanders, war machines)
-		if(bonus)
+		if(peculiarBonus)
```

Comment:
Does if/elseif structure means that we can have only one bonus either peculiar or additive or fixed? Because now if both peculiar and additive bonuses exist we will have only peculiar applied

Reply:
At first I didnt test combination of both bonuses, so made an "else" clause to make sure nothing bad happens.
But since you pointed this out, I changed it, tested it and it works: gave Labetha both bonuses in config, and they added to each other.